lib/animina/narratives/resources/story.ex

```diff
@@ -164,6 +167,14 @@ defmodule Animina.Narratives.Story do
     identity :unique_position, [:position, :user_id]
   end
 
+  changes do
+    change after_action(fn changeset, record, _ ->
+             update_user_registration_completed_at(record.user_id)
+             {:ok, record}
+           end),
+           on: [:create, :destroy]
```

Comment:
Why on `:destroy`?

Reply:
I had previously asked.
If I go an delete all my stories , my account becomes inactive doesn't it?

Reply:
Can I delete all stories? AFAIK I can not delete all but one story or can I?

Reply:
I maybe wrong here but AFAIK we implemented a rule which says that there always has to be an `About me` story.

Reply:
The rule we had was...you cannot delete the 'About me' story...if it the last one remaining, so if the last one remaining is not a an 'About me" story , then we can delete it and have no stories at all...should we change this rule?

Reply:
Of course. We always want to have at least one `About me` story. That is the minimum.

Reply:
Someone can have one story per headline . So they have one `About me` story . This new rule will mean , the user cannot delete all their stories , they have to remain with at least one?

Reply:
Yes, the `About me` story is the corner stone. A profile has to have at least this story.